tutorials/chapter1_annexB.md

```diff
@@ -0,0 +1,513 @@
+<p align="right">
+  <img width="25%" height="25%"src="../images/klepsydra_logo.jpg">
+</p>
+
+# Chapter 1 - Annex B: Publisher and subscriber arguments
+
```

Comment:
I suggest to add a Table of Contents (like in this file, for example: https://github.com/klepsydra-technologies/kpsr-tutorial/pull/35/files#diff-0946029d7f66bbd1fbb9a8f4473729200780611c9fadd038b964cab8cdf53814R7)